analytic_engine/src/table/data.rs

```diff
@@ -111,6 +111,28 @@ impl TableShardInfo {
     }
 }
 
+/// `atomic_enum` macro will expand method like
+/// ```text
+/// compare_exchange(..) -> Result<TableStatus, TableStatus>
+/// ```
+/// The result type is conflict with outer
+/// Result, so add this hack
+// TODO: fix this in atomic_enum crate.
+mod hack {
```

Comment:
Maybe there is no need to import atomic_enum, one AtomicU8 and three constants will be enough?